app/index.js

```diff
@@ -111,3 +117,23 @@ function onError(error) {
   }
 
 }
+
+// emitted when new client connects
+function onConnection(){
+  server.getConnections(function(error,count){
+    //console.log('Number of concurrent connections to the server : ' + count);
+    connections.set(count);
+  });
+}
+
+//Prometheus server configuration
+var metrics_server = http.createServer(function (request, response) {
+  response.writeHead(200, {'Content-Type': promClient.register.contentType});
+  response.end(promClient.register.metrics());
+});
+
+metrics_server.listen(9095, () => {
```

Comment:
This should not be a separate server.  It should just be an endpoint on the existing Express app we have already.

Reply:
Removed the prometheus server, added a /metrics endpoint on graphql server. listen on same 3333 port.  http://localhost:3333/metrics